### ISSUE TITLE
[1.8.0] @stencil-redux no longer supported, build fails

### ISSUE THREAD

**Issue by stanley85** [ionic-team/stencil#2015]
**Stencil version:**
```
 @stencil/core@1.8.0
```

**I'm submitting a:**
[X] bug report

**Current behavior:**
Updating from 1.7.5 to 1.8.0 breaks compile for stencil-redux

**Expected behavior:**
Please add some ionic love to stencil-redux and up the core - its 0.18

**Steps to reproduce:**
Add stencil-redux to a new ^1.8.0 project

**Other information:**
TypeScript
Import declaration conflicts with local declaration of 'Action'.

**./node_modules\@stencil\redux\dist\types\global\interfaces.d.ts**

_import { Action, AnyAction, Store as ReduxStore, Unsubscribe } from 'redux';
export interface Store<S = any, A extends Action = AnyAction> {_

**Comment by cmaas** [comment]
Yeah, two things though:

1. Could you open the issue in the `stencil-redux` repo? https://github.com/ionic-team/stencil-redux/issues

2. It's a very small fix really. You can do this by yourself in the meantime to have your project compiled until there is a proper bugfix release. Since `d.ts`-files aren't compiled anyways, it doesn't really matter that you simply modify the file as a workaround:

```
import { Action as ReduxAction, AnyAction, Store as ReduxStore, Unsubscribe } from 'redux';
export interface Store<S = any, A extends ReduxAction = AnyAction> {
```

Basically, tell this file that the "original" `Action` is a `ReduxAction` and a new `Action` is exported.

**Comment by stanley85** [comment]
Editing the local stencil-redux package wont help because of deployment procedures, but we are fine with 1.7.5 for now. Its a great stable release to reside on for while. Except for the latest 1.8.1 fix which could help fix the IE bug not showing the component when another stencil app resides in the dom.

**Comment by adamdbradley** [comment]
This should be fixed in the latest version. If it's still a problem, would you be able to create an issue with a thorough description on how to replicate this? Thanks
